packages/ethereum-storage/src/ethereum-utils.ts

```diff
@@ -1,50 +1,46 @@
-import { StorageTypes } from '@requestnetwork/types';
+import { CurrencyTypes, StorageTypes } from '@requestnetwork/types';
 import * as config from './config';
 
 import { BigNumber } from 'ethers';
+import { EVM } from '@requestnetwork/currency';
 
-const networks = {
-  [StorageTypes.EthereumNetwork.PRIVATE]: 'private',
-  [StorageTypes.EthereumNetwork.MAINNET]: 'mainnet',
-  [StorageTypes.EthereumNetwork.KOVAN]: 'kovan',
-  [StorageTypes.EthereumNetwork.RINKEBY]: 'rinkeby',
-  [StorageTypes.EthereumNetwork.GOERLI]: 'goerli',
-  [StorageTypes.EthereumNetwork.SOKOL]: 'sokol',
-  [StorageTypes.EthereumNetwork.XDAI]: 'xdai',
-};
+/**
+ * Collection of utils functions related to Ethereum Storage
+ */
 
 /**
- * Collection of utils functions related to Ethereum
+ * Get the name of the Ethereum network from its id
+ *
+ * @param networkId Id of the network
+ * @return name of the network
  */
-export default {
```

Comment:
I removed the default export here (cc @MantisClone) and exported these utilities in the index file of the package. It allows other packages to filter storage chains amongst all supported EVM chains.